util/internal/integrationtest/v5migrationhelper/src/main/java/org/hibernate/search/test/util/impl/ExpectedLog4jLog.java

```diff
@@ -10,21 +10,21 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
-import org.apache.log4j.AppenderSkeleton;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
-import org.apache.log4j.spi.LoggingEvent;
+import org.hibernate.search.util.impl.test.rule.log4j.LogProgrammatic;
+
+import org.junit.rules.TestRule;
+import org.junit.runners.model.Statement;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.LogEvent;
 import org.hamcrest.CoreMatchers;
 import org.hamcrest.Description;
 import org.hamcrest.Matcher;
 import org.hamcrest.StringDescription;
 import org.hamcrest.TypeSafeMatcher;
-import org.junit.rules.TestRule;
-import org.junit.runners.model.Statement;
 
 /**
```

Comment:
You didn't replace this version of `ExpectedLog4jLog` with the other one, as we discussed during the previous review?

Reply:
I've thought about it. I would prefer to change the `v5 migration helper` modules as little as possible. If there was something to make sharing, I would do it as a different contribution/issue.